example/advection/parthinput.advection

```diff
@@ -51,8 +53,12 @@ integrator = rk2
 cfl = 0.45
 vx = 1.0
 vy = 1.0
+vz = 1.0
+profile = hard_sphere
```

Comment:
Below the profiles are enumerated 0 through 2. And here it's a string. Am I missing something? Also string would definitely be better.

Reply:
Originally I didn't want to do string comparisons within a device kernel.
Now that all this runs on the host I can actually use the strings here.